doc/api/process.md

```diff
@@ -2044,6 +2044,42 @@ false
 
 See the [TTY][] documentation for more information.
 
+## process.throw(error)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `error` {any}
+
+Fatally exit as if the provided error value was thrown, using the regular fatal
```

Comment:
"the regular fatal error print logic" probably doesn't mean anything to most users.